src/internal/Subject.ts

```diff
@@ -14,15 +14,15 @@ import { arrRemove } from './util/arrRemove';
  * Subject, and you can call next to feed values as well as error and complete.
  */
 export class Subject<T> extends Observable<T> implements SubscriptionLike {
-  observers: Observer<T>[] = [];
-
   closed = false;
-
-  isStopped = false;
-
-  hasError = false;
-
-  thrownError: any = null;
+  /** @internal */
+  _observers: Observer<T>[] = [];
```

Comment:
There are a **LOT** of people that are checking `subject.observers.length === 0` or `> 0`, for better or worse. It's rare that people are checking anything other than whether or not the subject is being observed. I don't think we can rename `observers` to `_observers` without deprecating it first.

We may want to add a feature like `isObserved` or something like that on our Subjects, given how often I've seen code that needed to do that, because whether I agree with the need or not, other people have found the need for this.

Reply:
What is the way around this now, as i cannot seem to find an isObserved method?